src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -58,10 +58,43 @@ public void ConfigureServices(IServiceCollection services)
     services.AddOpenTelemetry()
         .WithTracing(builder => builder
             .AddAspNetCoreInstrumentation()
-            .AddJaegerExporter());
+            .AddOtlpExporter());
 }
 ```
 
+#### Metrics
+
+The following example demonstrates adding ASP.NET Core instrumentation with the
+extension method `AddOpenTelemetry().WithMetrics()` on `IServiceCollection`
+then extension method `AddHttpClientInstrumentation()` on `MeterProviderBuilder`
+to the application. This example also sets up the OTLP (OpenTelemetry Protocol)
+Exporter, which requires adding the package
+[`OpenTelemetry.Exporter.OpenTelemetryProtocol`](../OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md)
+to the application.
+
+```csharp
+using Microsoft.Extensions.DependencyInjection;
+using OpenTelemetry.Metrics;
+
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddOpenTelemetry()
+        .WithMetrics(builder => builder
+            .AddAspNetCoreInstrumentation()
+            .AddOtlpExporter());
+}
+```
+
+#### List of metrics produced
+
+The instrumentation was implemented based on [metrics semantic
```

Comment:
```suggestion
The instrumentation is implemented based on [metrics semantic
```

Reply:
Done 71301b6fd888981222fc2b3aa4d64116bacc7f08 together with the second occurence.